app/[locale]/tag/listing-tags.tsx

```
@@ -0,0 +1,126 @@
+    { value: "popularity", label: "Popularity" },
+    { value: "name-asc", label: "Name A-Z" },
+    { value: "name-desc", label: "Name Z-A" },
+    { value: "date-desc", label: "Newest" },
+    { value: "date-asc", label: "Oldest" },
+  ];
+
+  return (
+    <div className="text-center mb-12 py-6 container mx-auto px-4 mt-8">
+      <div className="flex items-center justify-center mb-6">
+        <div className="flex items-center text-gray-900 dark:text-gray-200 bg-gray-200 dark:bg-[#1F2937] py-2 px-4 rounded-full gap-2 text-sm font-medium">
+          <div className="w-2 h-2 bg-yellow-400 rounded-full animate-pulse"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)